video/out/hwdec/hwdec_cuda.h

```diff
@@ -50,10 +50,17 @@ struct cuda_mapper_priv {
     void *ext[4];
 };
 
-typedef bool (*cuda_interop_init)(const struct ra_hwdec *hw);
+typedef struct cuda_interop_fn {
+    bool (*check)(const struct ra_hwdec *hw);
+    bool (*init)(const struct ra_hwdec *hw);
+} cuda_interop_fn;
```

Comment:
I would remove typedef and reference it as `struct cuda_interop_fn` later. 

```suggestion
struct cuda_interop_fn {
    bool (*check)(const struct ra_hwdec *hw);
    bool (*init)(const struct ra_hwdec *hw);
};
```

Reply:
Done with https://github.com/mpv-player/mpv/pull/14028/commits/f234f91bf09fdb1c5af171d0ef6733a6a5eba67b